python/src/aiconfig/default_parsers/palm.py

```diff
@@ -338,10 +341,14 @@ async def run_inference(
         response = palm.chat(**completion_data)
         outputs = []
         for i, candidate in enumerate(response.candidates):
+            # candidate is a MessageDict obj (https://shorturl.at/jKY35), 
+            # but Pydantic TypedDict breaks for Python v<3.12  so using
+            # generic Dict type
+            candidate: Dict
```

Comment:
If there's anything other than `candidate.content` in `candidate`, then we should store it in the metadata